util/llpcEmuLib.h

```diff
@@ -42,6 +42,20 @@ class Function;
 class Module;
 } // llvm
 
+namespace std
```

Comment:
I'm not sure about this section.  I don't know how you want the hash functor to be defined.  If you have preferences let me know.